src/main/assisted-installer-controller/assisted_installer_main_test.go

```
@@ -157,19 +158,23 @@ var _ = Describe("installer HostRoleMaster role", func() {
 
 var _ = Describe("installer HostRoleMaster role agent-based installation", func() {
 	var (
-		l             = logrus.New()
-		ctrl          *gomock.Controller
-		mockk8sclient *k8s_client.MockK8SClient
-		kubeNamesIds  map[string]string
+		l                              = logrus.New()
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gci)